pkgs/development/libraries/miopen/default.nix

```diff
@@ -36,15 +35,11 @@
 , useOpenCL ? false
 }:
 
-assert buildDocs -> texlive != null;
-assert buildDocs -> doxygen != null;
-assert buildDocs -> sphinx != null;
-assert buildDocs -> python3Packages != null;
 assert buildTests -> zlib != null;
 assert fetchKDBs -> fetchurl != null;
```

Comment:
We also do not require this. The packages are not being build when the boolean is false and are always defined.